src/backend/PostgresBackend.cpp

```diff
@@ -484,7 +484,7 @@ PostgresBackend::fetchTransactions(
                     << ripple::strHex(hash) << "\'";
 
                 auto res = pgQuery(sql.str().data(), yield);
-                if (size_t numRows = checkResult(res, 4))
+                if (checkResult(res, 4))
```

Comment:
Can we use 17 style `if statement w/ initializer`s here?
```cpp
if (auto res = pgQuery(sql.str().data(), yield); checkResult(res, 4))
{
```

Reply:
Done